content/docs/user-guide/project-structure/pipelines-files.md

```diff
@@ -224,7 +224,7 @@ ${param.list[0]} # List elements via index in [] (square brackets)
 > To use the expression literally in `dvc.yaml` (so DVC does not replace it for
 > a value), escape it with a backslash, e.g. `\${...`.
 
-## Generating multiple stages
+## Foreach stages
```

Comment:
Does everyone agree on this term? Cc @dberenbaum

Reply:
`foearch` should be code-inlined I think ... There was an option like "Something (`foreach`)" - I would personally prefer that one

Reply:
I can't find any suggestion like "Something (`foreach`)" but I'm fine with "`foreach` stages"

Reply:
I think the "Something" was "Loop" but I argued we should avoid that direct reference.

Went for quoted **`foreach` stages** — a minor issue is how it looks in the ToC nav:

![image](https://user-images.githubusercontent.com/1477535/106346376-45b6b280-627c-11eb-9d92-d5b653abbf03.png)